xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/controller/ConsentControllerTest.java

```diff
@@ -52,30 +61,33 @@
     @Mock
     private HeadersMapper headersMapper;
 
+    @Spy
+    private ObjectMapper objectMapper;
+
     @BeforeEach
     void setUp() {
         MockitoAnnotations.initMocks(this);
 
         mockMvc = MockMvcBuilders.standaloneSetup(controller)
-            .setMessageConverters(new MappingJackson2HttpMessageConverter())
-            .setControllerAdvice(new RestExceptionHandler(new HeadersMapper()))
-            .build();
+                          .setMessageConverters(new MappingJackson2HttpMessageConverter())
```

Comment:
Is the Editor/Code Style/Java/Scheme set to Default in your preferences? Indentation here and on line 83 doesn't look right.

Reply:
The preferences should be fine, I must have added extra tab there. Fixed